Remind_Device/Remind_Device.ino

```diff
@@ -80,30 +85,68 @@ void setup() {
 
 void loop() { 
 
+  // State machine will Print out which state it moves to. Helps with troubleshooting
   // Starts state machine
   switch(currentState){
   
+      
     //=============================
     //    State One
     //=============================
 
     // starts state one of state machine
     case STATE_GENERAL_WAIT:
-
-      // If the joystick or button is pressed, move to STATE_LCD_ON
-      // If the motion detector is sensed, set the speaker counter to 1 and move to STATE_SPEAKER_PLAY
       
-      ConfirmationButtonState = digitalRead(ConfirmationButton);
+      // Print trouble shooting information 
+        // If button is pressed, print button pushed. - note: logic is inverted because of pullup input
+        if (digitalRead(ConfirmationButton) == LOW){
```

Comment:
This was all just me testing with the button, not sure if it should be the final place for a button to be pressed. I can test again if this method of and if statement works. It looks like it should